Created Splash Page:

stecd opened the pull request:
1) Oliver wrote a new index and css file
2) Stephanie edited some of oliver's code so that it worked more responsively
4) updated video background to have no text
3) Twitter link still has to be updated

Commit:
Created Splash Page:
1) Oliver wrote a new index and css file
2) Stephanie editted some of olivers code so that it worked more responsively
3) Twitter link still has to be updated

tinataleb commented on index-0.html:
```diff
@@ -0,0 +1,254 @@
+<!--
```
can you rename this file to something like "v1index" so we know this was the first version?

tinataleb commented on index.css:
```diff
@@ -0,0 +1,211 @@
+#content {
```
Can you move this file to assets/css?

tinataleb commented on index.html:
```diff
@@ -31,224 +18,44 @@
   <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:400,700|Material+Icons" />
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/latest/css/font-awesome.min.css">
   <!-- CSS Files -->
+  <link href="index.css" rel="stylesheet" />
```
After moving index.css to assets/css this has to be updated.

Commit:
Made Edits from PR
1) updated v1index name
2) moved css file to css folderMade Edits from PR
1) updated v1index name
2) moved css file to css folderMade Edits from PR
1) updated v1index name
2) moved css file to css folderMade Edits from PR
1) updated v1index name
2) moved css file to css folderMade Edits from PR
1) updated v1index name
2) moved css file to css folderMade Edits from PR
1) updated v1index name
2) moved css file to css folderMade Edits from PR
1) updated v1index name
2) moved css file to css folderMade Edits from PR
1) updated v1index name
2) moved css file to css folderMade Edits from PR
1) updated v1index name
2) moved css file to css folderMade Edits from PR
1) updated v1index name
2) moved css file to css folder

tinataleb commented:
The animation plays on a loop, my impression was that it should play once and stop at the end without going dark. 
Is this an expected behavior?

D-Arceneaux commented:
added in the C19.png photo for the web browser photo integration.
Code is there, .png missing

stecd commented:
@tinataleb yeah, Dylan and I liked the looping so we left it. but feel free to change it!
